common.gypi

```diff
@@ -566,6 +573,42 @@
           'OPENSSL_NO_ASM',
         ],
       }],
+      ['OS == "zos"', {
+        'defines': [
+          '_XOPEN_SOURCE_EXTENDED',
+          '_UNIX03_THREADS',
+          '_UNIX03_WITHDRAWN',
+          '_UNIX03_SOURCE',
+          '_OPEN_SYS_SOCK_IPV6',
+          '_POSIX_SOURCE',
+          '_OPEN_SYS',
+          '_OPEN_SYS_IF_EXT',
+          '_OPEN_SYS_SOCK_IPV6',
+          '_OPEN_MSGQ_EXT',
+          '_LARGE_TIME_API',
+          '_ALL_SOURCE',
+          '__IBMCPP_TR1__',
+          'NODE_PLATFORM="os390"',
+          'PATH_MAX=1024',
+        ],
+        'cflags': [
+          '-q64',
+          '-Wc,DLL',
+          '-Wa,GOFF',
+          '-qARCH=10',
+          '-qASCII',
+          '-qTUNE=10',
```

Comment:
Do you want to consider TUNE to a newer hardware level?

Reply:
we thought about to move the ARCH and TUNE to 11 but decided to keep it as 10 as V2R4 still support EC12

Reply:
For Java, we set ARCH to min hardware support, but TUNE to a more recent hardware that we expect most users to be on when they use the product.   Not sure how much of a difference that makes tho.

Reply:
I build with just setting TUNE to 11 and notice not much performance difference (slight increase in acmeair and decrease in octane), do you think we should change TUNE to 11? @IgorTodorovskiIBM

Reply:
How large was the perf increase in acmeair?  Also, is it reproducible?  Curious if TUNE(12) gives better results

Reply:
The increase is ~2% and seems to be consistent after a couple run, I also tried TUNE(12) but the result is the same (~2% improvement)

Reply:
I change it to 12